core/kazoo_json/test/kz_json_test.erl

```diff
@@ -55,18 +56,15 @@ prop_get_value() ->
                       end)
            ).
 
--define(DO_NOT_RUN_QC_SET_VALUE, true).
--ifndef(DO_NOT_RUN_QC_SET_VALUE).
```

Comment:
This test was disabled because it sometimes failed in CI. How are you fixing it?